src/App.tsx

```diff
@@ -14,6 +18,10 @@ function App() {
     <div className="App">
       <h1>Change money</h1>
       {ListCurrencies(allCurrencies)}
+      <div>
+          <input type="text" onClick={() => setReversed(prev => !prev)} onChange={(e) => setFirstInputValue(e.target.value)}/>
```

Comment:
What happens if I will clicks only for first input ?

Reply:
You will change the flag(isReversed)
*click = true
*click = false 
Yes, if user will click two times on the same input that our converter will not work correctly(if we look to the future)

Reply:
fix it